.idea/gitql.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Thank you for the change but you commit some IDE files :/ . Can you remove them?